src/main.zig

```diff
@@ -1,11 +1,13 @@
 const std = @import("std");
+const util = @import("util.zig");
 const print = std.debug.print;
 const EnumField = std.builtin.Type.EnumField;
 const fmt = std.fmt;
-
-const EVM = @import("evm.zig").EVM;
+const DummyEnv = util.DummyEnv;
+const EVM = util.EVM;
```

Comment:
I am bringing your attention to the fact that this is no longer importing the EVM type from `evm.zig` but the one instantiated using the dummy environment in `util.zig`. Maybe it should be renamed `Test` or `NoOp`.